### PR TITLE
Add more error reporting for getGradleSourceSets failure

### PR DESCRIPTION
**Pull request by Arthurm1**
This passes `stderr` back if retrieving source sets fails.  Before there sometimes wasn't enough information to diagnose.

### CODE REVIEW

File: server/src/main/java/com/microsoft/java/bs/core/internal/gradle/GradleApiConnector.java [hunk 1]
```diff
@@ -93,10 +96,10 @@ public GradleSourceSets getGradleSourceSets(URI projectUri) {
       // since the model returned from Gradle TAPI is a wrapped object, here we re-construct it
       // via a copy constructor and return as a POJO.
       return new DefaultGradleSourceSets(customModelBuilder.get());
-    } catch (GradleConnectionException | IllegalStateException e) {
-      summary = e.getMessage();
+    } catch (GradleConnectionException | IllegalStateException | IOException e) {
+      summary = e.getMessage() + "\n" + errorOut.toString();
```
Comment (jdneo) [inline server/src/main/java/com/microsoft/java/bs/core/internal/gradle/GradleApiConnector.java]: Let's check if errorOut is empty or not before appending to the summary text.

Comment (Arthurm1) [reply server/src/main/java/com/microsoft/java/bs/core/internal/gradle/GradleApiConnector.java]: Done

**Review by jdneo — APPROVED** [review]
LGTM

**Comment by jdneo** [comment]
Thank you @Arthurm1

**Pull request merged**